lib/screens/register_screen.dart

```diff
@@ -17,181 +20,272 @@ class RegisterScreen extends StatefulWidget {
 
 class RegisterScreenState extends State<RegisterScreen> {
   bool _showSpinner = false;
-  String _name, _email, _password, _confirmPassword, _address;
+  String _name, _email, _password, _confirmPassword;
   Firestore _firestore = Firestore.instance;
 
+  void _openAddressModal(AddressModel addressModel) async {
+    showModalBottomSheet(
+      clipBehavior: Clip.antiAlias,
+      enableDrag: false,
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(20.0),
+          topRight: Radius.circular(20.0),
+        ),
+      ),
+      context: context,
+      builder: (context) {
+        return AddressSelector(addressModel: addressModel);
+      },
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
-    return SafeArea(
-      child: Scaffold(
-        resizeToAvoidBottomInset: true,
-        body: ModalProgressHUD(
-          color: kPrimaryColor,
-          inAsyncCall: this._showSpinner,
-          child: Material(
-            child: Container(
-              color: kBackgroundColor,
-              child: Center(
-                child: NotificationListener<OverscrollIndicatorNotification>(
-                  onNotification: (OverscrollIndicatorNotification overscroll) {
-                    overscroll
-                        .disallowGlow(); // This will stop overscroll glow effect.
-                    return;
-                  },
-                  child: ListView(
-                    padding:
-                        EdgeInsets.symmetric(horizontal: 50.0, vertical: 10.0),
-                    shrinkWrap: true,
-                    children: <Widget>[
-                      Center(
-                        child: Stack(
-                          children: <Widget>[
-                            Text(
-                              'Start',
-                              style: kHeadingStyle,
+    return ChangeNotifierProvider<AddressModel>(
+      create: (context) => AddressModel(),
+      child: SafeArea(
+        child: Scaffold(
+          resizeToAvoidBottomInset: true,
+          body: ModalProgressHUD(
+            color: kPrimaryColor,
+            inAsyncCall: this._showSpinner,
+            child: Material(
+              child: Container(
+                color: kBackgroundColor,
+                child: Padding(
```

Comment:
Remove this padding widget.
Use `padding:`  given in `ListView()`  widget itself. 
This will allow user to scroll even from padded area.

Also add some vertical padding, as there is a no space between last line 'Already a member? Login!' and bottom screen.

Reply:
Done